src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  message: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    message: 'Nothing was pressed yet',
+  };
+
+  handleClick = (event: KeyboardEvent) => {
```

Comment:
Let's fix this problem
![image](https://user-images.githubusercontent.com/96409563/213114550-b51c684e-f963-4450-8ceb-b816b308919e.png)